src/kyc-verification/utils/file-encryption.util.ts

```
@@ -0,0 +1,75 @@
+   * @returns Object containing the file path and encryption key
+   */
+  static async encryptAndSave(
+    buffer: Buffer,
+    destPath: string,
+  ): Promise<{ filePath: string; encryptionKey: string }> {
+    // Generate encryption key and initialization vector
+    const key = crypto.randomBytes(32);
+    const iv = crypto.randomBytes(16);
+
+    // Create cipher
+    const cipher = crypto.createCipheriv(this.algorithm, key, iv);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)